test/libsolidity/semanticTests/strings/concat/string_concat_2_args.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+    function f(string memory a, string memory b) public returns (string memory) {
+        return string.concat(a, b);
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// f(string,string): 0x40, 0x80, 32, "abcdabcdabcdabcdabcdabcdabcdabcd", 5, "bcdef" -> 0x20, 0x25, 0x6162636461626364616263646162636461626364616263646162636461626364, 44502269928904312298000709931354278973409164155382318144318241583783949107200
```

Comment:
The expectation formatting here is unfortunate - we may want to file an ``isoltest`` issue about that, if there is not already... for ``bytes memory`` it works better.
Nothing we need to do in this PR, though.